src/js/components/stock-movement-wizard/combined-shipments/SendMovementPage.jsx

```diff
@@ -420,7 +421,12 @@ class SendMovementPage extends Component {
           values: {
             ...this.state.values,
             dateShipped: stockMovementData.dateShipped,
-            shipmentType: _.get(stockMovementData, 'shipmentType.id'),
+            shipmentType: {
+              id: stockMovementData.shipmentType.id,
```

Comment:
I think you could here simply destructure `stockMovementData.shipmentType` and add only a label field since it is  made with `splitTranslation` (you might need to remove the property of `labelKey` from field configuration), and on top of that I think you don't have to add value field, since it's defined in the field config as `valueKey`.

(applies to all 3 files)